### PR TITLE
enhancement: added total count for selected status and removed the view button for application card

### LINKED ISSUE #859: Enhancement : There is an alignment issue with the new elements that are created in the applications page.
### Issue Description

Currently there is an alignment issue with the elements
### Reproducibility

 - [ ] This issue is reproducible
 - [ ] This issue is not reproducible

### Steps to Reproduce
1. 
2. 

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [ ] Medium
 - [ ] Low

### Checklist

- [ ] I have read and followed the project's code of conduct.
- [ ] I have searched for similar issues before creating this one.
- [ ] I have provided all the necessary information to understand and reproduce the issue.
- [ ] I am willing to contribute to the resolution of this issue.

### PR DESCRIPTION
**Pull request by rishirishhh**
Date: 25/09/2024

Developer Name: Kavya Rishi 

## Issue Ticket Number

- Closes #859 

## Description
This contains the frontend changes which remove the View-Details button from the application card and whenever you hover over the application card the cursor changes to pointer and then superuser can click on the application card to view more details. This also contains the change which shows the total number of accepted/pending/rejected applications as per the selected filter.
### Documentation Updated?

- [ ] Yes
- [X] No

### Under Feature Flag

- [X] Yes
- [ ] No

### Database Changes

- [ ] Yes
- [X] No

### Breaking Changes

- [ ] Yes
- [X] No

### Development Tested?

- [X] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

### Before: 
Screenshot from 2024-09-25 18-09-57

Screenshot from 2024-09-25 18-12-25

### After: 
Screenshot from 2024-09-28 12-00-51

</details>

## Test Coverage
Screenshot from 2024-09-27 00-16-40

### CODE REVIEW

**Review by vinit717 — CHANGES_REQUESTED** [review]
Tests are missing

**Comment by shubhdevelop** [comment]
Please add screenshot of the test coverage in PR description
and screencast of changes made, and how it's behaving

File: __tests__/applications/applications.test.js
Comment (joyguptaa) [inline __tests__/applications/applications.test.js:170]: make this test name short and precise

File: applications/script.js
Comment (joyguptaa) [inline applications/script.js:347]: Do we have test for totalCountElement?

Comment (rishirishhh) [reply applications/script.js:347]: Yes there is a test that checks the existence of totalCountElement

File: __tests__/applications/applications.test.js
Comment (rishirishhh) [inline __tests__/applications/applications.test.js:181]: @joyguptaa  Here we check for the existence

**Review by joyguptaa — DISMISSED** [review]

**Review by Achintya-Chatterjee — DISMISSED** [review]

**Review by Achintya-Chatterjee — DISMISSED** [review]

**Review by joyguptaa — DISMISSED** [review]

**Review by yesyash — DISMISSED** [review]

**Comment by vinit717** [comment]
Please confirm the UI with Ankush as the alignment is not good

File: applications/index.html
Comment (yesyash) [inline applications/index.html:1]: the cards, title and filter button do not seem to be aligned please fix this.

**Review by yesyash — CHANGES_REQUESTED** [review]

File: applications/index.html
Comment (rishirishhh) [reply applications/index.html:1]: I have confirmed the UI with Ankush and he approves the new changes

**Comment by vinit717** [comment]
You have removed the view details button now how does the super user view the details of the application?

**Comment by rishirishhh** [comment]
> You have removed the view details button now how does the super user view the details of the application?

I have removed the view details behind the dev flag 
behind the dev flag the user can simply click on application modal 
whenever you hover around the application card , cursor changes into pointer and you can click on it
@vinit717

**Comment by rishirishhh** [comment]
[Screencast from 28-09-24 01:29:53 PM IST.webm](https://github.com/user-attachments/assets/b387ca96-e9ae-4585-89a5-909203ed68bd)
@vinit717

**Comment by vinit717** [comment]
> [Screencast from 28-09-24 01:29:53 PM IST.webm](https://github.com/user-attachments/assets/b387ca96-e9ae-4585-89a5-909203ed68bd) @vinit717

Cursor is not visible and text is not align with the card yet

**Comment by rishirishhh** [comment]
> > [Screencast from 28-09-24 01:29:53 PM IST.webm](https://github.com/user-attachments/assets/b387ca96-e9ae-4585-89a5-909203ed68bd) @vinit717
> 
> Cursor is not visible and text is not align with the card yet

cursor is not visible in screencast , I cannot do anything about that

**Comment by vinit717** [comment]
> > > [Screencast from 28-09-24 01:29:53 PM IST.webm](https://github.com/user-attachments/assets/b387ca96-e9ae-4585-89a5-909203ed68bd) @vinit717
> > 
> > 
> > Cursor is not visible and text is not align with the card yet
> 
> cursor is not visible in screencast , I cannot do anything about that

Sorry But I cannot approve it without the proof

**Comment by rishirishhh** [comment]
https://github.com/Real-Dev-Squad/website-dashboard/issues/859 issue to be resolved after this merge

File: applications/script.js
Comment (vinit717) [inline applications/script.js:374]: why is the CSS in the js file?

**Review by yesyash — APPROVED** [review]

**Review by Achintya-Chatterjee — APPROVED** [review]

**Review by vinit717 — APPROVED** [review]

**Pull request merged**
